src/index.ts

```diff
@@ -11,15 +10,11 @@ export {
   LoaderError,
 } from './lazy';
 
-export type {
-  ClientLoader,
-  Loader,
-  Options as LazyOptions,
-  ServerLoader,
-} from './lazy';
+export { ClientLoader, Loader, LazyOptions, ServerLoader } from './lazy';
```

Comment:
This should also export `LazyComponent`, I must've missed it in initial implementations since Jira is using flow types

Reply:
@MonicaOlejniczak I have exported the `LazyComponent` as well from index file and force pushed. Thanks for catching that. I experimented with CRA latest version and it doesn't seem like CRA handles `export type` syntax without errors 'cos it's on TS v3.7.2.

Reply:
What type of warnings do you run into in Jira?

Reply:
When you don't use the `type` keyword, the build output contains references to type exports in the library files (e.g. `dist/esm/index.js`). These type exports cannot be resolved as they are stripped out during the build. Here's an example:

```
// esm/index.js
export { ClientLoader, Loader, LazyOptions, ServerLoader } from './lazy';

// esm/lazy/index.js
export { Asset, Manifest, LazyOptions, LazyComponent };

// esm/lazy/types.js
// empty file -- LazyComponent and LazyOptions do not exist

```